waku/waku_core/subscription.nim

```diff
@@ -0,0 +1,7 @@
+import
```

Comment:
The changes in `waku/waku_filter/client.nim`, and the addition of the `subscription` module, I think can go in a separate PR because it is a refactoring. That way, we will make this PR simpler :)